src/command/explain.rs

```diff
@@ -0,0 +1,25 @@
+use crate::command::RoverStdout;
+use crate::error::metadata::code::Code;
+use crate::Result;
+use serde::Serialize;
+use structopt::StructOpt;
+
+#[derive(Debug, Serialize, StructOpt)]
+pub struct Explain {
+    /// The full error code. For example, E020
+    #[structopt(name = "CODE")]
+    code: Code,
+}
+
+impl Explain {
+    pub fn run(&self) -> Result<RoverStdout> {
+        let explanation = &self.code.explain();
+
+        // if we're printing all codes, we don't need to pretty print them
+        if self.code == Code::EALL {
+            Ok(RoverStdout::PlainText(explanation.clone()))
```

Comment:
Definitely agree it's easiest to look up one at a time. I'm just trying to provide a mechanism to be able list them all for a docs page for example. 

Although maybe we could just scan the `codes` directory, and load the md files in a build script, and use the filenames as headings for them. Thoughts on that, and dropping the `EALL` case entirely @lrlna @EverlastingBugstopper?

Reply:
oh, I am down to have an API endpoint that would list them all for docs! I just don't know if we need it as part of the CLI. Or were you going to use the CLI to add it to the docs? If that's the case, then definitely just leave it here.

Reply:
I was planning on using the CLI for it, but that would still mean it being a manual release step, since the CLI has to be built to run that command 😅 It may actually just be simpler in the long run to make a build step that does that